packages/wrangler/src/config/index.ts

```diff
@@ -254,14 +254,26 @@ export function printBindings(bindings: CfWorkerInit["bindings"]) {
 		});
 	}
 
-	if (unsafe !== undefined && unsafe.length > 0) {
-		output.push({
-			type: "Unsafe",
-			entries: unsafe.map(({ name, type }) => ({
-				key: type,
-				value: name,
-			})),
-		});
+	if (unsafe !== undefined) {
+		if (unsafe.bindings.length > 0) {
+			output.push({
+				type: "Unsafe Bindings",
```

Comment:
Can we keep this as `Unsafe` to avoid any potential breaking change?

Reply:
Done.